boards/e180-zg120b-tb/Kconfig

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+config BOARD
+    default "e180-zg120b-tb" if BOARD_E180_ZG120B_TB
+
+config BOARD_E180-ZG120B-TB
+    bool
+    default y
+    select BOARD_COMMON_SILABS
+    select CPU_MODEL_EFR32MG1B232F256GM32
+    select HAS_PERIPH_ADC
+    select HAS_PERIPH_RTC
+    select HAS_PERIPH_RTT
+    select HAS_PERIPH_TIMER
+    select HAS_PERIPH_UART
+    select HAS_PERIPH_UART_MODECFG
```

Comment:
`HAS_PERIPH_UART_MODECFG` should be a CPU feature

Reply:
If added to the CPU features, doesn't that automatically add a dependency on the UART peripheral?

Reply:
No, it just means that the CPU implements this feature.
This way we can extend the API without having to implement it for all platforms at once.